ayon_api/server_api.py

```diff
@@ -7785,13 +7789,10 @@ def get_entities_links(
 
         Args:
             project_name (str): Project where links are.
-            entity_type (Literal["folder", "task", "product",
-                "version", "representations"]): Entity type.
-            entity_ids (Optional[Iterable[str]]): Ids of entities for which
-                links should be received.
+            entity_type (Literal["folder", "task", "product", "version", "representations"]): Entity type.
+            entity_ids (Optional[Iterable[str]]): Ids of entities for which links should be received.
```

Comment:
Too long lines.

Reply:
Adjusted (in the code and in my IDE), thanks :)